x-pack/plugins/ingest_manager/server/errors/handlers.ts

```diff
@@ -48,6 +61,21 @@ export const defaultIngestErrorHandler: IngestErrorHandler = async ({
   response,
 }: IngestErrorHandlerParams): Promise<IKibanaResponse> => {
   const logger = appContextService.getLogger();
+  if (isLegacyESClientError(error)) {
+    // there was a problem communicating with ES (e.g. via `callCluster`)
+    // only log the message
+    logger.error(error.message);
+
```

Comment:
Why not log the more elaborate `message` that is constructed below, and passed to `response.customError()`?

Reply:
No real reason not to. I considered it at one point. Happy to update it.

Reply:
I think it would be helpful to have the same level of detail in both the log and the returned message.

Reply:
Totally. Just pushed https://github.com/elastic/kibana/pull/76865/commits/fdaf5883cb7646601b98cb844bdfc79860bd0042

Reply:
Thank you!